### PR TITLE
repositories: add hikari overlay

### PR DESCRIPTION
**Pull request by AtharvaAmritkar**
Hello,I would like to add my overlay of the programs I work on

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -5126,6 +5126,18 @@
     <source type="git">ssh://git@git.zx2c4.com/portage</source>
     <feed>http://git.zx2c4.com/portage/atom/?h=master</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+      <name>hikari</name>
```
Comment (jsmolic) [inline files/overlays/repositories.xml:5130]: Please keep the list sorted

**Review by jsmolic — CHANGES_REQUESTED** [review]

File: files/overlays/repositories.xml (hunk 1 above)
Comment (AtharvaAmritkar) [reply files/overlays/repositories.xml]: > Please keep the list sorted

I have sorted them.

Comment (jsmolic) [reply files/overlays/repositories.xml]: Thanks

File: files/overlays/repositories.xml [hunk 2]
```diff
@@ -2072,6 +2072,18 @@
     <feed>https://cgit.gentoo.org/user/hibiscus.git/atom/</feed>
     <!-- <feed>https://cgit.gentoo.org/user/hibiscus.git/rss/</feed> -->
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>hikari</name>
+    <description lang="en">hikari's overlay</description>
+    <homepage>https://github.com/AtharvaAmritkar/hikari-overlay</homepage>
+    <owner type="person">
+      <email>atharvaamritkar@protonmail.com</email>
```
Comment (jsmolic) [inline files/overlays/repositories.xml:2080]: Have you opened a bugzilla account with this e-mail, I can't find it on bugs.gentoo.org

Comment (AtharvaAmritkar) [reply files/overlays/repositories.xml:2080]: Yes, could you please recheck

Comment (AtharvaAmritkar) [reply files/overlays/repositories.xml:2080]: is it not in the list?

Comment (jsmolic) [reply files/overlays/repositories.xml:2080]: Yes, I see it now (there was only the gmail-one previously)

**Review by jsmolic — APPROVED** [review]
Thanks, I'm going to squash and merge

**Pull request closed without merging**